src/react/deckgl.js

```diff
@@ -27,7 +27,16 @@ const {Deck, log} = experimental;
 
 const propTypes = Object.assign({}, Deck.propTypes, {
   viewports: PropTypes.array, // Deprecated
-  viewport: PropTypes.object // Deprecated
+  viewport: PropTypes.object, // Deprecated
+
+  // Viewport props (TODO - should only support these on the react component)
+  longitude: PropTypes.number, // The longitude of the center of the map.
```

Comment:
Should we extract these from `viewports` prop when it is present. Does application has to provide these for transitions to work?